media/live-seekable-range.html

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I'm not sure where that script error was coming from.
Also, I confirmed I had set my M60's chrome://flags to enable experimental web platform features (which is why set/clearLiveSeekableRange is working for me). I'm sorry for that bit of review noise :/